packages/eslint-config-custom/index.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
         ],
       },
     ],
+    '@typescript-eslint/no-floating-promises': 'warn',
```

Comment:
Maybe info until they are fixed we might miss other important warns otherwise

Reply:
Sadly 'info' doesn't exist for eslint, only 'off', 'warn' and 'error' 😞